src/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module "svelte-collapse" {
+  import type { ActionReturn } from "svelte/types/runtime/action";
+
+  interface CollapseParams {
+	open: boolean;
+	duration: number;
+	easing: string;
+  }
+
+  export default function collapse(node: HTMLElement, params: CollapseParams): ActionReturn;
```

Comment:
I think that this should be made `Partial<CollapseParams>` so that the object is not required to have all the properties. Defaults are used internally to fill any missing properties.

Reply:
makes sense! done :)